### ISSUE TITLE
tomllib is not considered as in standard library in Format imports

### ISSUE THREAD

**Issue by GuillaumeLeGoc** [astral-sh/ruff-vscode#465]
Hi,
I'm using vscode on a project with a conda environment using python 3.12. The conda environment is selected in vscode. When running `Ruff: Format imports` from the command palette, `tomllib` is moved with non-standard imports.

Before `Ruff: Format imports` :
```python
import os
import tomllib

import numpy as np
```

After `Ruff: Format imports` :
```python
import os

import numpy as np
import tomllib
```

The issue was raised here https://github.com/astral-sh/ruff/issues/2343, and supposedly fixed here https://github.com/astral-sh/ruff/pull/2345.

System information :
- Windows 10
- vscode 1.89.1
- Ruff vscode extension v2024.20.0

Do I have to take further action to fix the problem (configuration, ...) ?

Cheers !

**Comment by dhruvmanila** [comment]
I think this will only work if the required Python version is 3.11 or greater as `tomllib` was added in that Python version. Can you share your `pyproject.toml`? You could also run `ruff` from the command-line with the `--verbose` flag which will provide which category an import is in:

```
[2024-05-20][11:53:16][ruff_linter::rules::isort::categorize][DEBUG] Categorized 'tomllib' as Known(StandardLibrary) (KnownStandardLibrary)
```

**Comment by charliermarsh** [comment]
👍 You can add something like:

```toml
[project]
...
requires-python = ">=3.11"
```

...to your `pyproject.toml`, to indicate that your project only supports Python 3.11 and later. Alternatively, you can tell Ruff to treat 3.11 as your minimum supported version with:

```toml
[tool.ruff]
target-version = "py311"
```

**Comment by charliermarsh** [comment]
Closing for now, let me know if you're continuing to see this after applying the above.

**Comment by hobbsd** [comment]
Ruff is sorting tomllib outside standard lib even though I have in my pyproject.toml:
```
[project]
...
requires-python = ">=3.13"
```
This is within VSCode v1.96.2 with the Ruff v2024.56.0 extension.
